src/java/org/apache/cassandra/transport/CQLMessageHandler.java

```diff
@@ -454,6 +526,30 @@ protected String id()
         return channel.id().asShortText();
     }
 
+    private void scheduleConnectionWakeupTask(long waitLengthMicros)
```

Comment:
It might be slightly preferable to pass the units alongside, so that the caller is forced to ensure they consider the units they provide.